Черанева Ольга

OlgaCheraneva opened the pull request:
[Посмотреть решение](https://hruview.herokuapp.com/OlgaCheraneva/markup-task-8/master/index.html)

OlgaCheraneva committed:
Решение (+extra)

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

vaseker commented:
🍅 
Хорошее начало.
А теперь анимации надо переписать на translate3d/scale3d/rotate3d таким образом, чтобы не было repaint'ов страницы и fps не скакал, да и вообще был повыше.

<img width="1176" alt="2018-11-27 21-20-46" src="https://user-images.githubusercontent.com/1942271/49095537-81175a00-f28a-11e8-8265-84f54a8339d6.png">

OlgaCheraneva committed:
Исправления

Не получается анимировать включение света в окнах поезда без repaint. Псевдоэлементы для compartment-window уже использованы, поэтому я не могу задать им желтый фон и анимировать opacity, что не вызовет repaint. Другого способа решения этой проблемы я не могу найти

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

vaseker commented on index.css:
```diff
@@ -272,12 +278,12 @@ ul
     from
     {
-        right: -44rem;
+        transform: scale(.99) translate3d(44rem, 0, 0);
```
scale3d

vaseker reviewed (changes requested):
🍅

vaseker commented:
🍏 
Хотя отличный результат
